backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -42,6 +42,31 @@
 TRACIM_DEFAULT_PERM = "tracim"
 
 
+class TracimSecurityPolicy:
+    """
+    Adapter to keep existing authentification method compatible with pyramid 2.0
+    source:
+    https://docs.pylonsproject.org/projects/pyramid/en/latest/whatsnew-2.0.html#upgrading-from-third-party-policies
+    """
+
+    def __init__(self, authn_policy, authz_policy):
+        self.authn_policy = authn_policy
+        self.authz_policy = authz_policy
```

Comment:
Maybe more explicit naming would help understand?
And should the policies be configurable? Or would it be enough to instantiate them here?

Reply:
Yes, it's possible to change the naming.
The policies are configurable and instantiated before running into this code, in fact, we instantiale multiple authentication policy into a single multiauth policy and a "allow all" authorization policy as we handle authorization with check_right decorator that doesn't use pyramid specific code.

Reply:
renaming done.